### PR TITLE
Fixed all instances of calls silently dropping params

### PR DESCRIPTION
**Pull request by arvicco**
…, added type coercion to order params

### CODE REVIEW

File: lib/bitfinex/orders.rb [hunk 1]
```diff
@@ -18,6 +18,9 @@ module OrdersClient
     def new_order(symbol, amount, type, side, price = nil, params = {})
       check_params(params, %i{is_hidden is_postonly ocoorder buy_price_oco})
 
+      # for 'market' order, we need to pass a random positive price, not nil
+      price ||= rand(100) if type == "market" || type == "exchange market"
```
Comment (prdn) [inline lib/bitfinex/orders.rb]: You may just force `price = 0.00001` for market orders

**Comment by prdn** [comment]
@arvicco thanks for all the great code you're pushing
Just a comment for you

File: lib/bitfinex/orders.rb (hunk 1 above)
Comment (arvicco) [inline lib/bitfinex/orders.rb]: Yeah, it actually makes sense. I was following the API doc too literally. ;)

**Comment by prdn** [comment]
Looks good

**Pull request merged**